src/components/CharacterList.js

```diff
@@ -1,14 +1,23 @@
-import React from "react";
+import React from 'react';
 
-import Character from "./Character";
+import Character from './Character';
 
 const CharacterList = props => {
+  // I spent a very long time tryng to debug this project. It turns out it was set up wrong in the first place. In addition to a change in reducers/index.js, it needed 'props.characters.characters.map' on line 11:
   return (
-    <ul>
-      {props.characters.map(character => {
-        return <Character key={character.name} character={character} />;
-      })}
-    </ul>
+    <div>
+      {props.characters && (
```

Comment:
I know Dustin used this in his demo, but well done using this syntax (called a `short circuit evaluation`) instead of a longer if statement